new/LIT7150Miracle.xml

```diff
@@ -0,0 +1,143 @@
+                        This file is licensed under the Creative Commons
+                        Attribution-ShareAlike 4.0. 
+
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <abstract>
+                <p>
+                    Yǝsḥaq, a devoted monk under Bishop Gabriel of Alexandria, dedicated himself
+                    to fervent nightly prayers before the Virgin Mary's icon for seven years, bowing
+                    down one hundred times each night and imploring Jesus to reveal His mother to him.
+                    On the night of the feast of the 21st of Tāḥsās, the Virgin Mary appeared to him,
+                    acknowledging his prolonged suffer-ing and assuring him that God had heard his
+                    prayers. She promised that he would join her in three days, which came true as
+                    Yǝsḥaq peacefully passed away three days later and was re-ceived into her presence.
+                </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                    <term key="Religion"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="GS" when="2024-11-28">Created entity</change>
+            <change who="GS" when="2024-11-28">Added relations and description</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT7150Miracle" passive="LIT2384Taamme"/>
+                   <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="AkkMir6A"
+                        corresp="LIT5673AkkonuColl"/>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle"
+                        passive="LIT3592Miracle"/>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle"
+                        passive="LIT7151Miracle"/>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML5520">
+                        <desc>see <ref type="mss" corresp="EMML5520">EMML 5520, f. 53r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML 8828">
```

Comment:
as already written, there may be no blanks inside of URIs or IDs

Reply:
for manuscripts - do not list them in relations, list them as `witness `in `listWitness`